api/pkg/transformer/symbol/function/geospatial.go

```diff
@@ -55,45 +55,45 @@ func (l *LatLong) toS2ID(level int) interface{} {
 
 // Geohash convert latitude longitude to geohash value
 func Geohash(lat interface{}, lon interface{}, precision uint) (interface{}, error) {
-	latLong, err := extractLatLong(lat, lon)
+	ll, err := extractLatLong(lat, lon)
 	if err != nil {
 		panic(err)
 	}
 
-	switch latLong.(type) {
+	switch latLong := ll.(type) {
 	case []*LatLong:
 		var value []interface{}
-		for _, ll := range latLong.([]*LatLong) {
+		for _, ll := range latLong {
```

Comment:
the alternating usage from ll to latlong and back to ll here is pretty confusing and may lead to potential bugs next time. Can we have more meaningful name? Or at least a different name for the inner ll?

Reply:
Good point. Updated it.